src/sentry/static/sentry/app/views/performance/landing/vitalsCards.tsx

```diff
@@ -32,6 +39,15 @@ import {
 } from '../vitalDetail/utils';
 import VitalPercents from '../vitalDetail/vitalPercents';
 
+import {backendCardDetails, getBackendFunction} from './utils';
+
+const Sparklines = React.lazy(
+  () => import(/* webpackChunkName: "Sparklines" */ 'app/components/sparklines')
+);
+const SparklinesLine = React.lazy(
+  () => import(/* webpackChunkName: "SparklinesLine" */ 'app/components/sparklines/line')
+);
```

Comment:
Does bundling these into the view add a sizable weight? It seems like the only thing we're deferring is the rendering of the charts which should be cheaper/faster than loading the data. Should the data loading also be deferred?

Reply:
I've removed the lazy loading for the chart.